process/hyper_block_processor.go

```diff
@@ -6,25 +6,31 @@ import (
 )
 
 type hyperBlockProcessor struct {
-	transactionProcessor TransactionHandler
-	shardBlocksProcessor ShardBlocksHandler
+	transactionProcessor    TransactionHandler
+	shardBlocksProcessor    ShardBlocksHandler
+	epochStartInfoProcessor EpochStartInfoHandler
 }
 
 // NewHyperBlockProcessor will create a new instance of an hyper block processor
 func NewHyperBlockProcessor(
 	transactionHandler TransactionHandler,
 	shardBlockHandler ShardBlocksHandler,
+	epochStartInfoProcessor EpochStartInfoHandler,
```

Comment:
epochStartInfoHandler to keep the pattern used for the other parameters

Reply:
Right, missed it, good catch